hdk-rust/src/api/link_entries.rs

```diff
@@ -3,10 +3,18 @@ use error::ZomeApiError;
 use holochain_core_types::cas::content::Address;
 use holochain_wasm_utils::api_serialization::link_entries::LinkEntriesArgs;
 
-/// Adds a named, directed link between two entries on the DHT.
-/// Consumes three values, two of which are the addresses of entries, and one of which is a string that defines a
-/// relationship between them, called a `tag`. Later, lists of entries can be looked up by using [get_links](fn.get_links.html). Entries
+/// Adds a named, tagged, directed link between two entries on the DHT.
```

Comment:
This is a great description, we also need to get this version into the mdbook part of the docs, not just here...